controllers/resultController.js

```diff
@@ -50,7 +50,7 @@ const deleteResultById = async (req, res, next) => {
 const findByIdAndUpdate = async (req, res, next) => {
   try {
     const id = req.params.resultId;
-    const model = { ...req };
+    const model = { ...req.body };
```

Comment:
this won't work. Since we are not expecting model on body object when request is made